uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/ContactActionsMenu/Page/Inline/ActionsPart1.tpl

```diff
@@ -0,0 +1,90 @@
+<div class="ac_results" id="crm-contact-actions-list">
+  <div id="bootstrap-theme">
+    <div class="row show-grid">
```

Comment:
What is the purpose of `show-grid` class?

Reply:
Left after I comped the markup fon style guide. No need here. will remove.